agent/battle/vision/prompts.py

```
@@ -0,0 +1,27 @@
+"""多模态视觉 Prompt 构造。"""
+from __future__ import annotations
+
+import json
+
+from .models import VisionRequest
+
+
+SYSTEM_PROMPT = """你是 MaaFgo 战斗界面的视觉分析器。只分析截图，不执行点击，不输出坐标，不制定战术。
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)